pkg/proxy/winkernel/metrics.go

```diff
@@ -27,10 +27,19 @@ const kubeProxySubsystem = "kubeproxy"
 
 var (
 	SyncProxyRulesLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Subsystem: kubeProxySubsystem,
+			Name:      "sync_proxy_rules_latency_seconds",
+			Help:      "SyncProxyRules latency in seconds",
+			Buckets:   prometheus.ExponentialBuckets(0.001, 2, 15),
+		},
+	)
+
+	DeprecatedSyncProxyRulesLatency = prometheus.NewHistogram(
 		prometheus.HistogramOpts{
 			Subsystem: kubeProxySubsystem,
 			Name:      "sync_proxy_rules_latency_microseconds",
-			Help:      "SyncProxyRules latency",
+			Help:      "SyncProxyRules latency in microseconds",
```

Comment:
Should we output the deprecation notice in the help?

Reply:
Great idea. This kind of notice should be applied in all deprecation metrics. I will mark all of them later.